apps/challenges/views.py

```diff
@@ -1840,6 +1843,12 @@ def download_all_submissions(
                             submission["created_at"],
                             submission["submission_result_file"],
                             submission["submission_metadata_file"],
+                            ",".join(
+                                f'{key}: {value}'
+                                for key, value in submission[
+                                    "submission_metadata"
+                                ].items()
+                            ),
```

Comment:
@afif-fahim can we move this to line 1813.

Reply:
@Ram81, do you mean before `writer.writerow`?